zephyr/panic.c

```diff
@@ -1,9 +1,13 @@
 // SPDX-License-Identifier: BSD-3-Clause
 //
 // Copyright(c) 2019 Intel Corporation. All rights reserved.
-//
-// Author: Tomasz Lauda <tomasz.lauda@linux.intel.com>
 
```

Comment:
Please do not remove existing authors!

Reply:
there is nothing left from the original file